boot/zephyr/pm.yml

```diff
@@ -0,0 +1,33 @@
+# The size of this partition is defined by the kconfig symbol
+# CONFIG_PM_PARTITION_SIZE_MCUBOOT
+mcuboot:
+  placement:
+    # MCUboot must be placed in front of 'mcuboot_pad'.
+    before: [mcuboot_pad]
+
+mcuboot_partitions:
+  # Define a set of sub-partitions which spans over 'mcuboot_pad', 'spm' and 
+  # 'app' if both are present. If only 'app' is present, then these partitions
+  # will only span across that partition.
+  # The sub partitions share the size of the parent partition(s) equally.
+  sub_partitions: [primary, secondary]
+  span: [mcuboot_pad, spm, app]
+
+# The size of this partition is defined by the kconfig symbol
+# CONFIG_PM_PARTITION_SIZE_MCUBOOT_SCRATCH
+mcuboot_scratch:
+  placement:
+    after: [app]
```

Comment:
For my own understanding. Would it work to express "after: [mcuboot_partitions]" ?

Reply:
It should work. Have not tested this yet.